content/docs/concepts/multicluster-deployments/index.md

```diff
@@ -100,16 +102,16 @@ same IP address.
 In this topology, the Istio control plane is deployed on one of the clusters while all other
 clusters run a simpler remote Istio configuration which connects them to the single Istio control plane
 that manages all of the Envoy's as a single mesh. The IP addresses on the various clusters must not
-overlap and DNS resolution for services on remote clusters is not automatic.  Users need to replicate
+overlap and DNS resolution for services on remote clusters is not automatic. Users need to replicate
 the services on every participating cluster.
 
 To configure this type of multicluster topology, visit our
-[single control plane with VPN instructions](/docs/setup/kubernetes/install/multicluster/vpn/).
+[single-network shared control plane instructions](/docs/setup/kubernetes/install/multicluster/shared-vpn/).
 
-#### Single control plane without VPN connectivity
+#### Multi-network shared control plane
```

Comment:
for a followup - the key thing to explain about multi-network is that the networks are isolated, not that a gateway is necessary.  A gateway is always necessary if the networks are isolated.  I am not sure we want to use the terminology "Isolated" though - it has negative connotations for the title.  For explanation text, it would be very valuable.